bedrock/firefox/views.py

```diff
@@ -59,6 +67,10 @@
 ]
 STUB_VALUE_RE = re.compile(r'^[a-z0-9-.%():_]+$', flags=re.IGNORECASE)
 
+UNFCK_EMAIL_FROM = 'Mozilla.com <noreply@mozilla.com>'
+UNFCK_EMAIL_SUBJECT = 'Tell the world what you wanna unfck'
+UNFCK_EMAIL_TO = ['TODO@mozilla.com']
```

Comment:
Who will be the poor soul to wade into all these emails?

Reply:
Still TBD.